connectors/class-connector-posts.php

```diff
@@ -24,8 +26,9 @@ class Connector_Posts extends Connector {
 	 * @var array
 	 */
 	public $actions = array(
-		'transition_post_status',
 		'deleted_post',
+		'post_updated',
```

Comment:
<!-- ideas that popped up from reviewing -->
**💡 thought:** I believe this will not log new posts creation because `post_updated` is [only called for existing posts](https://github.com/WordPress/wordpress-develop/blob/6.6/src/wp-includes/post.php#L4860-L4860).

It may be a good thing because I think we don't need to log e.g. a `post_title` change for a new post. However, without `transition_post_status` callback we no longer log new posts creation. I guess we could use e.g. [`save_post`](https://github.com/WordPress/wordpress-develop/blob/6.6/src/wp-includes/post.php#L4891) action or bring back `transition_post_status` but limit its scope to only log transitions to the `publish` status.

Reply:
@delawski `$update` is [a check on whether or not the data has a `post_id`](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/post.php#L4254) which gets set immediately in the editor but ... if someone is using `wp_insert_post()` directly to add a post then it won't so I need to add that in, thank you!

Reply:
@delawski I've changed this to use `wp_after_insert_post` -- it's noisier but because of the orders of the filters in the REST API, it's the one that works there as well as in the Quick Edit section.

I still need to test and add automated tests though!